chore(internal): update internal deps

davidkonigsberg opened the pull request:
## Description
Updated internal deps to re-generated version from newer generators

## Changes Made
- generator-exec-sdk
- fiddle-sdk
- snippet-idk

## Testing
CI

davidkonigsberg committed:
update generator-exec-sdk

davidkonigsberg committed:
update snippet-sdk

davidkonigsberg committed:
Update fiddle-sdk

devin-ai-integration[bot] committed:
fix: Update LicenseConfig visitor usage for new generator-exec-sdk

The new version of generator-exec-sdk changed the LicenseConfig type to use a namespace-based pattern where _visit is a static method rather than an instance method. Updated the code to use FernGeneratorExec.LicenseConfig._visit() with explicitly typed parameters.

Co-Authored-By: thomas@buildwithfern.com <tjb9dcshop@gmail.com>

devin-ai-integration[bot] committed:
fix: Use switch statement for LicenseConfig instead of visitor pattern

The updated generator-exec-sdk v0.0.1169 changed the LicenseConfig type structure and no longer includes the visitor pattern (_visit method). Updated to use a simpler switch statement approach to handle the union type.

Co-Authored-By: thomas@buildwithfern.com <tjb9dcshop@gmail.com>

devin-ai-integration[bot] committed:
fix: Update Ruby codegen for generator-exec-sdk v0.0.1169

- Removed invalid import of BasicLicense and CustomLicense from generator-exec-sdk/api
- Replaced visitor pattern usage with switch statement for LicenseConfig
- Fixed EndpointIdentifier to use identifier_override (snake_case) instead of identifierOverride
- Fixed EndpointSnippet.ruby() constructor call to use object literal with type field

Co-Authored-By: thomas@buildwithfern.com <tjb9dcshop@gmail.com>

devin-ai-integration[bot] committed:
fix: update browser-compatible-base for generator-exec-sdk v0.0.1169

- Remove invalid imports from @fern-fern/generator-exec-sdk/api and /serialization
- Replace _visit pattern with switch statements for OutputMode, LicenseConfig, and registry configs
- Use _type instead of type for union discriminators
- Fix GeneratorEnvironment property access (_type, coordinatorUrl)

Co-Authored-By: thomas@buildwithfern.com <tjb9dcshop@gmail.com>

devin-ai-integration[bot] committed:
fix: update GeneratorsConfiguration for generator-exec-sdk v0.0.1169

- Replace _visit pattern with switch statements for OutputMode and PublishInfo
- Handle GithubV2 separately from Github due to schema differences

Co-Authored-By: thomas@buildwithfern.com <tjb9dcshop@gmail.com>

devin-ai-integration[bot] committed:
fix: update buildGeneratorConfig test files for generator-exec-sdk v0.0.1169

- Replace OutputMode.github() with object literal using type discriminator
- Replace GeneratorEnvironment.local() with object literal using _type discriminator
- Replace OutputMode.publish() and GeneratorPublishTarget.pypi() with object literals
- All test files now use explicit object literals instead of removed helper methods

Co-Authored-By: thomas@buildwithfern.com <tjb9dcshop@gmail.com>

devin-ai-integration[bot] committed:
fix: update generator base files for generator-exec-sdk v0.0.1169

- Remove GeneratorExecParsing import (no longer exported)
- Replace GeneratorUpdate static methods with object literals using _type discriminator
- Replace ExitStatusUpdate static methods with object literals
- Fix identifierOverride -> identifier_override in AbstractReadmeSnippetBuilder

Co-Authored-By: thomas@buildwithfern.com <tjb9dcshop@gmail.com>

tjb9dc commented on generators/browser-compatible-base/src/utils/getPackageName.ts:
```diff
@@ -1,32 +1,56 @@
-                _other: () => undefined
-            }),
-        _other: () => {
-            throw new Error("Unrecognized output mode.");
```
This is a behavior change. We want to still throw if we don't recognize the output mode. Try not to introduce behavior changes during upgrades of a wire SDK

tjb9dc commented on generators/base/src/utils/parseGeneratorConfig.ts:
```diff
@@ -1,20 +1,9 @@
-    }
-
-    return parsedConfig.value;
+    return JSON.parse(configStr.toString()) as FernGeneratorExec.GeneratorConfig;
```
you shouldn't have had to make these changes. You should be able to access the `parse` function from the serialization package in the SDK

devin-ai-integration[bot] committed:
fix: update typescript-v2 buildGeneratorConfig for generator-exec-sdk v0.0.1169

- Replace OutputMode.publish() with object literal
- Replace GeneratorPublishTarget.npm() with object literal using type discriminator

Co-Authored-By: thomas@buildwithfern.com <tjb9dcshop@gmail.com>
